rust/tests/BUILD

```diff
@@ -36,6 +36,7 @@ rust_test(
         "@crates//:cucumber",
         "@crates//:futures",
         "@crates//:regex",
+        "@crates//:serde_json",
```

Comment:
Serde-json was already used by Cloud, we repurpose it here as a JSON parser for behaviour tests.